src/PerformanceTests/Persistence.V5.Azure_v6/AzureProfile.cs

```diff
@@ -1,9 +1,22 @@
-﻿using NServiceBus;
+﻿using System;
+using NServiceBus;
+using NServiceBus.Persistence;
+using NServiceBus.SagaPersisters;
+using Tests.Permutations;
+using Variables;
 
-public class AzureProfile : IProfile
+public class AzureProfile : IProfile, INeedPermutation
 {
-    public void Configure(BusConfiguration busConfiguration)
+    public Permutation Permutation { private get; set; }
+
+    public void Configure(BusConfiguration cfg)
     {
-        busConfiguration.UsePersistence<AzureStoragePersistence>();
+        if (Permutation.OutboxMode == Outbox.On) throw new NotSupportedException("Outbox is not supported with Azure storage.");
```

Comment:
If INeedPermutation is set and Permutation is null, throw exception?

`if (this is INeedPermutation && Permutation == null) throw new InvalidOperationException("Cowabunga dude, you forgot to set Permutation property.");`